web/src/scripts/containers/ProjectNavigator.jsx

```diff
@@ -104,17 +108,20 @@ class ProjectNavigator extends Component {
 
   handlePreviewClick = async (node) => {
     const {rootPath, textEditorCompositeActions} = this.props
+    this.setState({currentPreview: node.path})
     const relativePath = node.path.replace(rootPath, '.')
     const requireText = relativePath.replace(path.extname(relativePath), '')
     textEditorCompositeActions.updateDecoEntryRequire(requireText)
   }
 
-  renderNode = (props) => {
+  renderNode = (props, state) => {
```

Comment:
probably you didn't mean to pass in this state, cause state isn't passed to this function, and you're using this.state below